PHP/PHP Source.sublime-syntax

```diff
@@ -1905,17 +1905,17 @@ contexts:
 
 ###[ TYPE HINTS ]#############################################################
 
-  type-hints:
+  type-hints-push:
```

Comment:
I'd prefer to keep the plural = pushing vs. singular = setting/popping style of context naming rather than adding arbitary `set`, `pop`, ... pre- or suffixes. 

`-pop` suffix is dedicated to those prototype like contexts which contain patterns which are re-used to pop contexts ( like `else-pop` ...).

Reply:
... otherwise we end up adding `set` and `pop` or `push` to nearly each context, which would add much naming repetition.

Reply:
ok. i didn't aware this naming convention at all. so get confused.